<launch>
	
	 <node name="qbo_stereo_selector" pkg="qbo_stereo_selector" type="qbo_stereo_selector" output="screen" >
	 	<param name="move_head" value="true" />
                <param name="search_min_pan" value="-0.8" />
                <param name="search_max_pan" value="0.8" />
                <param name="search_pan_vel" value="0.3" />
                <param name="search_min_tilt" value="-0.7" />
                <param name="search_max_tilt" value="-0.4" /> 
     	<param name="roi_mean_threshold" value="60" /> 
     	<param name="match_threshold_cam_shift" value="70" /> 
     	<param name="undetected_threshold" value="10" /> 
     	<param name="distance_filter_threshold" value="0.8" />      	
	 </node>


	 
</launch>
